src/js/leafletMap.js

```diff
@@ -0,0 +1,162 @@
+/* global L, fluid */
+
+"use strict";
+
+var hortis = fluid.registerNamespace("hortis");
+
+// Adapted from https://github.com/amb26/bagatelle/blob/master/src/client/js/leafletMap.js
+// Needs to be consolidated to produce a core implementation
+
+fluid.defaults("hortis.leafletMap", {
+    gradeNames: "fluid.viewComponent",
+    selectors: {
+        map: ".fld-mapviz-map",
+        tooltip: ".fld-mapviz-map-tooltip"
+    },
+    members: {
+        map: "@expand:L.map({that}.dom.map.0, {that}.options.mapOptions)"
+    },
+    events: {
+        buildMap: null
+    },
+    mapOptions: {
+        zoomSnap: 0.1
+    },
+    tileOptions: {
+        // tileUrl, tileAttribution
+    },
+    datasets: {},
+    markup: {
+        tooltip: "<div class=\"fld-mapviz-tooltip\"></div>",
```

Comment:
I know infusion uses `fl-` prefix for styling classes and `flc-` for component controlled classes. What is `fld-` prefix for? For holding data?

Reply:
I've tried to use fld for a prefix which is both used for styling as well as being component controlled. However I'm not sure that this prefixing system fits in with our latest thinking, since I often find that I am renaming prefixes back and forth as their role changes. Do we have a more up to date set of guidelines on CSS class names?

Reply:
The only guideline I know of about CSS naming conventions is [this wiki](https://wiki.fluidproject.org/display/fluid/Coding+and+Commit+Standards#CodingandCommitStandards-CSSNamingConventions) that was created years ago.

I don't know if there's one that's more up to date. @jobara, do you have an idea?